src/uct/api/uct.h

```diff
@@ -3000,11 +3000,15 @@ UCT_INLINE_API unsigned uct_iface_progress(uct_iface_h iface)
  * @param [in]  component   Component on which to open the connection manager,
  *                          as returned from @ref uct_query_components.
  * @param [in]  worker      Worker on which to open the connection manager.
+ * @param [in]  config      CM configuration options. Should be obtained
+ *                          from uct_cm_config_read() function, or point to
+ *                          CM-specific structure which extends uct_cm_config_t.
```

Comment:
Plz add ref for uct_cm_config_t

Reply:
@bbenton Added - https://github.com/openucx/ucx/pull/4574